### PR TITLE
Pass configmaps from API query to play kube

### PR DESCRIPTION
**Pull request by machacekondra**
This PR fixes the passing of the configmaps via the REST API, previously
the configmaps wasn't able to pass because the query with configmaps
wasn't parsed.

[NO NEW TESTS NEEDED]
[NO TESTS NEEDED]

Signed-off-by: Ondra Machacek <omachace@redhat.com>

When running code such as:
```golang
        manifestPath := "podyaml"
	configMaps := []string{"configmap.yaml"}
	options := play.KubeOptions{
		ConfigMaps: &configMaps,
	}
	report, err := play.Kube(connText, manifestPath, &options)
	if err != nil {
	  fmt.Printf("Error: %v", err)
	}
	fmt.Printf("Report: %v", report)

```

previously the configmaps wasn't passed, with this code it is.

### CODE REVIEW

**Comment by rhatdan** [comment]
/approve
LGTM
@umohnani8 @jwhonce PTAL

**Comment by umohnani8** [comment]
LGTM
Will wait on a final review from @jwhonce

**Review by Luap99 — CHANGES_REQUESTED** [review]
Please see https://github.com/containers/podman/pull/12371
The configmap should be added to the payload. Sending a path via API does not work when your client is on another system. I do not think we should add this to the API

**Comment by machacekondra** [comment]
> Please see #12371 The configmap should be added to the payload. Sending a path via API does not work when your client is on another system. I do not think we should add this to the API

Ah right, I think that this should be then removed from bindings? So it users are not confused. I am closing this PR.

**Comment by rhatdan** [comment]
Interested in opening a PR to remove from the bindings.
Should we even support remote podman play kube, until we fully suppor it remotely.
For example I am not sure --build works.

**Pull request closed without merging**
